StepLang/Examples/looping.step

```diff
@@ -6,3 +6,34 @@ while (i != 10) {
 }
 
 print("done", EOL)
+
+list l = [1, 2, 3, 4]
+
+// iterate over the list values
+foreach (number n in l) {
+	print(n, EOL)
+}
+
+// iterate over the list values and keys
+foreach (number k: number v in l) {
+	print(k, ": ", v, EOL)
+}
+
+map m = {"a": 1, "b": 2, "c": 3}
+
+// iterate over the map values
+foreach (number n2 in m) {
+	print(n2, EOL)
+}
+
+// iterate over the map values and keys
+foreach (string k2: number v2 in m) {
```

Comment:
wait… the syntax is different here than in the example above. is it `:` or `,`?

Reply:
Already fixed. It should be `:`